include/zephyr/bluetooth/audio/bap.h

```diff
@@ -41,9 +41,73 @@
 extern "C" {
 #endif
 
+#if defined(CONFIG_BT_BAP_BASS_MAX_SUBGROUPS)
+#define BT_BAP_BASS_MAX_SUBGROUPS CONFIG_BT_BAP_BASS_MAX_SUBGROUPS
+#else
+#define BT_BAP_BASS_MAX_SUBGROUPS 0
+#endif /* CONFIG_BT_BAP_BASS_MAX_SUBGROUPS*/
+
 /** An invalid Broadcast ID */
 #define BT_BAP_INVALID_BROADCAST_ID 0xFFFFFFFFU
 
+/**
+ * @name Suggested connection and advertising intervals suited for coexistence with audio
+ *
+ *
+ * Suggest use is to use this as both the minimum and maximum interval for best results.
+ * The intervals are suited for both unicast and broadcast.
+ * @{
```

Comment:
Chose to use a set of intervals in MS, rather than sets of extended advertising parameters, periodic advertising parameters and sets of connection parameters. 

If someone thinks the 3 types of sets (similar to how we have `BT_LE_CONN_PARAM_DEFAULT`, `BT_LE_ADV_CONN`, BT_LE_PER_ADV_DEFAULT, etc.), please let me know so we can discuss it, but it will be 3 times as many macros.